Resources/Prototypes/BLUESPACE/RHBZZ.yml

```
@@ -0,0 +1,249 @@
+    sprite: Clothing/Hands/Gloves/RHBZZ.rsi
+  - type: Clothing
+    sprite: Clothing/Hands/Gloves/RHBZZ.rsi
+  - type: Insulated
+  - type: Fiber
+    fiberMaterial: fibers-rubber
+    fiberColor: fibers-yellow
+  - type: FingerprintMask
+
+
+#униформа
+
+#униформа специалиста РХБЗЗ
+
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitRHBZZSpecialist
+  name: униформа специалиста РХБЗЗ
+  description: Он изготовлен из специального волокна, обеспечивающего особую защиту от биологических опасностей. На нем есть нашивка специалиста РХБЗЗ.
+  components:
+  - type: Sprite
+    sprite: Clothing/Uniforms/Jumpsuit/RHBZZ-jumpsuit.rsi
+  - type: Clothing
+    sprite: Clothing/Uniforms/Jumpsuit/RHBZZ-jumpsuit.rsi
+
+#униформа коммандера РХБЗЗ
+
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitRHBZZCommander
+  name: униформа коммандера РХБЗЗ
+  description: Он изготовлен из специального волокна, обеспечивающего особую защиту от биологических опасностей. На нем есть нашивка коммандера РХБЗЗ.
+  components:
+  - type: Sprite
+    sprite: Clothing/Uniforms/Jumpsuit/RHBZZ_leader-jumpsuit.rsi
+  - type: Clothing
+    sprite: Clothing/Uniforms/Jumpsuit/RHBZZ_leader-jumpsuit.rsi
+
+
+#берет
+
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHatBeretCommanderRHBZZ
+  name: берет коммандера РХБЗЗ
+  description: Выполненный в стилистике РХБЗЗ берет с эмблемой командирского звания.
+  components:
+  - type: Sprite
+    sprite: Clothing/Head/Hats/RHBZZ_leader-beret.rsi
+  - type: Clothing
+    sprite: Clothing/Head/Hats/RHBZZ_leader-beret.rsi
+  - type: Tag
+    tags:
+    - ClothMade
+    - HamsterWearable
+    - WhitelistChameleon
+
+
+#айди карта
+
+- type: entity
+  parent: CentcomIDCard
+  id: CentcomIDCardRHBZZ
+  name: айди карта РХБЗЗ
+  components:
+  - type: Sprite
+    layers:
+    - state: centcom
+  - type: Item
+    heldPrefix: blue
+  - type: IdCard
+    jobTitle: Centcom Agent
+
+
+#рюкзак РХБЗЗ
+
+- type: entity
+  parent: ClothingBackpack
+  id: ClothingBackpackRHBZZ
+  name: полевой рюкзак РХБЗЗ
+  description: Рюкзак, побывавший во многих опасных местах, надежный боевой рюкзак.
+  components:
+    - type: Sprite
+      sprite: Clothing/Back/Backpacks/merc.rsi
+  - type: Storage
+    grid:
+    - 0,0,6,3
+    maxItemSize: Huge
+
+
+
+#Куклы
+
+- type: randomHumanoidSettings
+  id: RHBZZSpecialist
+  components:
+    - type: MindShield
+    - type: GhostRole
+      name: Специалист РХБЗЗ
+      description: Устраняйте биологическую угрозу любой ценой!
+    - type: GhostTakeoverAvailable
+    - type: Loadout
+      prototypes: [ RHBZZSpecialistGear ]
+    - type: RandomMetadata
+      nameSegments:
+      - NamesFirstMilitaryLeaderCorvax # Corvax-Localization
+      - NamesLastERTCallsign # Corvax-Localization
+
+- type: randomHumanoidSettings
+  id: RHBZZCommander
+  components:
+    - type: MindShield
+    - type: GhostRole
+      name: Коммандер Эскадрона Смерти

```

```suggestion
      name: Командир РХБЗЗ
```
Мне кажется не эксадрон)